treeherder/webapp/api/performance_data.py

```diff
@@ -36,16 +36,16 @@ def get_performance_series_summary(self, request, project, jm, pk=None):
         """
         GET method implementation for listing signatures
 
-        Input: time interval
```

Comment:
You need to update the `_calculate_etag` method above to incorporate the `machine_platform` parameter. As it is now, the etag doesn't incorporate this information, so we will only ever cache one response to this method (and return the same value thereafter).